config/_default/menus/menus.en.yaml

```diff
@@ -1800,10 +1800,10 @@ main:
     parent: profiler_enabling
     identifier: profiler_enabling_php
     weight: 107
-  - name: Enabling the Linux Profiler
-    url: tracing/profiler/enabling/linux/
+  - name: Enabling the Compiled Languages Profiler
```

Comment:
Much clearer.  Total nit:  I recognize that the rest of our portfolio has product names of the form "X Profiler," but to my eye "Compiled Languages Profiler" reads awkwardly.  Is this improved by calling it the "Profiler for Compiled Languages" instead?

I don't have a strong opinion here, it's just that I have no idea whether this is just a matter of taste!  In English we have the royal order of adjectives, so I'm wondering whether my preference is something similar or just my own personal idiosyncrasy.

Either way, much better!

Reply:
Update:  I see elsewhere in this PR that we call this the "profiler for compiled languages."  So in addition to my question above, should we standardize on one or the other?